build-scripts/compile_all.py

```diff
@@ -105,6 +105,7 @@ def load_resolve_and_validate_profiles(env_yaml, profile_files, loader, controls
         p.validate_variables(loader.all_values.values())
         p.validate_rules(loader.all_rules.values(), loader.all_groups.values())
         p.validate_refine_rules(loader.all_rules.values())
+        p.unselect_empty_groups(loader.loaded_group)
```

Comment:
How about calling `unselect_empty_groups()` after we `load_compiled_content()` in `build_shorthand.py`?
Maybe in `load_benchmark()`?